docs/Rakefile

```diff
@@ -6,7 +6,12 @@ task :default => :gen
 BUILD_DIR          = "build"
 MARKDOWN_FILES     = Rake::FileList.new("source/_tmp/*.md")
 TOPIC_MAP          = "#{BUILD_DIR}/_topic_map.yml"
-ADOC_VARIABLES     = "source/variables.adoc"
+ADOC_VARIABLES     = FileList.new('source/variables.adoc',
+ 'source/contributing/variables.adoc',
+ 'source/getting-started/variables.adoc',
+ 'source/openshift/variables.adoc',
+ 'source/using/variables.adoc'
+)
```

Comment:
You broke it down because you thought there would be name/vale clash for variables?